content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -190,7 +188,8 @@ pod/reviews-v2-5b667bcbf8-spnnh labeled
 Any requests from pods in the ambient mesh to the `reviews-v2` pod IP will now be routed through the `reviews-v2-pod-waypoint` waypoint for L7 processing and policy enforcement.
 
 {{< tip >}}
-The original addressing of the traffic is used to determine if a service or workload waypoint should be used. Traffic which is addressed to a service, even though ultimately this would be resolved to a pod IP, is always treated by the ambient mesh as to-service and would use a service-attached waypoint. Workload-attached waypoints are only used when the client addresses traffic to the pod itself.
+The original destination type of the traffic is used to determine if a service or workload waypoint will be used. By using the original destination type the ambient mesh avoids having traffic transit waypoint twice, even if both service and workload have attached waypoints.
```

Comment:
We used `service` and `waypoint` (with backticks) above.  Worth being consistent?

Reply:
I don't think they should be backticked. We typically backtick commands and when discussing Kubernetes resources.

Reply:
I'll second that. In this use service and workload are being used simply to qualify the waypoint. In the previous use I did backtick them because those are possible arguments which can be used with --for which seemed to me like a different use.

This might be better still if I revise the wording to be:

...determine if a service-attached or workload-attached waypoint will...